main.js

```diff
@@ -0,0 +1,20 @@
+var icon = document.getElementById("icon");
+const button = document.querySelector("#startButton");
+
+
+button.addEventListener("click", logIn);
+function logIn() {
+  if (startButton.value !== "start") {
+    window.location.href = './home/home.html';
```

Comment:
Can you express me why you are using this conditional if statement here? 
What is it's function? and why you are need it for?

Reply:
Right, there's no need for it .